src/Layout/LayoutStyle.scss

```diff
@@ -2,48 +2,170 @@
 
 
 
-#cards-framework-container {
-  background-color: $layout-bg;
-  font-family: $font-stack;
+#cardsFrameworkContainer {
```

Comment:
interesting that we are using id for the container.

Reply:
that's the code previously... if we stumble across a defect where we have multiple instances of this component on the same view, we can address this then